### PR TITLE
Fix broken comments introduced by ddmd conversion

### PR DESCRIPTION
**Pull request by 9rnsr**
Broken style was introduced by ddmd conversion.

Ping: @yebblies

### CODE REVIEW

File: src/dinterpret.d [hunk 1]
```diff
@@ -1140,9 +1140,8 @@ public:
         {
             Statement sx = (*s.statements)[i];
             Expression e = interpret(sx, istate);
-            if (!e) // suceeds to interpret, or goto target
+            if (!e) // suceeds to interpret, or goto target was not fonnd
```
Comment (yebblies) [inline src/dinterpret.d]: `s/fonnd/found/`

Comment (9rnsr) [inline src/dinterpret.d]: I didn't notice the existing typo. Fixed.

**Comment by yebblies** [comment]
Auto-merge toggled on

**Comment by 9rnsr** [comment]
@yebblies Thanks!

**Pull request merged**